committing.rst

```diff
@@ -258,6 +258,41 @@ Developers can apply labels to GitHub pull requests).
 
 .. _cherry_picker.py: https://github.com/python/core-workflow/tree/master/cherry_picker
 
+.. _git_worktrees:
+
+Speeding up development workflow using git worktrees
+----------------------------------------------------
+
+When backporting changes to multiple versions, it's often useful to have more than one checkouts
+of the codebase. This way you can do things like running tests on one version
+while committing a patch on another version. For example: ::
```

Comment:
This paragraph seems to have lines that are too long.

I'd prefer to add two spaces after the full stops.

You can just use ``For example::`` -- it will automatically add a ``:`` and format the next block of code.